dotnet-client-libraries/HandlingDatesWithNodaTime/Tests/NodaTimeUnitTest.cs

```diff
@@ -0,0 +1,120 @@
+﻿using NodaTime;
+using NodaTime.Text;
+
+namespace Tests;
+
+public class NodaTimeUnitTest
+{
+    [Fact]
+    public void WhenGetCurrentInstantMethodIsCalled_ThenReturnCurrentInstant()
+    {
+        var instant = SystemClock.Instance.GetCurrentInstant();
+
+        Assert.True(instant != default(Instant));
+    }
+
+    [Fact]
+    public void WhenFromUnixTimeMethodIsCalled_ThenReturnCurrentInstant()
+    {
+        var instant = Instant.FromUtc(2023, 8, 31, 20, 00);
+
+        var expected = new LocalDateTime(2023, 8, 31, 20, 00)
+            .InZoneStrictly(DateTimeZone.Utc)
+            .ToInstant();
+
+        Assert.Equal(expected, instant);
+    }
+
+    [Fact]
+    public void WhenFromUnixTimeTicksMethodIsCalled_ThenReturnCurrentInstant()
+    {
+        var epochTime = 1693153840;
+
+        var instant = Instant.FromUnixTimeTicks(epochTime);
+        var unixTimeTicks = instant.ToUnixTimeTicks();
+
+        Assert.Equal(epochTime, unixTimeTicks);
+    }
+
+    [Fact]
+    public void WhenNumberOfTicksIsSpecified_ThenReturnEquivalentDuration()
+    {
+        var ticks = 600000000;
+
+        var duration = Duration.FromTicks(ticks);
+
+        Assert.Equal(ticks, duration.BclCompatibleTicks);
+    }
+
+    [Fact]
+    public void WhenNumberOfHoursIsSpecified_ThenReturnEquivalentDuration()
+    {
+        var hours = 12;
+
+        var duration = Duration.FromHours(hours);
+
+        Assert.Equal(hours, duration.Hours);
+    }
+
+    [Fact]
+    public void WhenLocalTimeIsInstantiated_ThenReturnLocalTime()
+    {
+        var localTime = new LocalTime(10, 30);
+
+        Assert.Equal(10, localTime.Hour);
+        Assert.Equal(30, localTime.Minute);
+    }
+
+    [Fact]
+    public void WhenLocalTimeIsConvertedToString_ThenReturnString()
```

Comment:
This should be renamed a bit also as we are not just checking that it is a string, but that it is being formatted in a specific way.

Reply:
I've made the change. Please take a look.